contracts/cw721-base/src/contract.rs

```diff
@@ -783,4 +805,103 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn approving_all_revoking_all() {
+        let mut deps = mock_dependencies(20, &[]);
+        setup_contract(&mut deps);
+
+        // Mint a couple tokens (from the same owner)
+        let token_id1 = "grow1".to_string();
+        let name1 = "Growing power".to_string();
+        let description1 = "Allows the owner the power to grow anything".to_string();
+        let token_id2 = "grow2".to_string();
+        let name2 = "Growing power".to_string();
```

Comment:
This should be slightly different that name1. Even calling it "Growing Power 2" is fine.